src/javascript/app/pages/user/account/settings/financial_assessment.js

```diff
@@ -49,7 +49,7 @@ const FinancialAssessment = (() => {
         }
 
         // display Trading Experience only for financial & MT5 financial accounts
-        const is_mt5_financial = /labuan_advanced|real_vanuatu_(standard|advanced|mamm_advanced)/.test(localStorage.getItem('financial_assessment_redirect'));
+        const is_mt5_financial = /labuan_advanced|real_svg_standard|real_vanuatu_advanced|real_vanuatu_mamm_advanced/.test(localStorage.getItem('financial_assessment_redirect'));
```

Comment:
@brandon-binary then we should also be able to remove it from here since FA is also not needed for svg